components/signinpage.tsx

```diff
@@ -0,0 +1,140 @@
+"use client"
+
+import React, { useState, ChangeEvent } from 'react';
+import { Passwordreveal } from '@/assets/icons';
+import { Googleicon } from '@/assets/icons';
+import { Facebookicon } from '@/assets/icons';
+import { Vectorline } from '@/assets/icons';
+
+
+
+const Signinpage = () => {
```

Comment:
I would like to suggest you to rename this component to `SigninForm` and rename this file to `signin-form.tsx` because there have a component which is in similar name.